etcdserver/server.go

```diff
@@ -62,7 +61,8 @@ const (
 	StoreKeysPrefix    = "/1"
 
 	purgeFileInterval      = 30 * time.Second
-	monitorVersionInterval = 10 * time.Second
+	monitorVersionInterval = 5 * time.Second
+	versionUpdateTimeout   = 1 * time.Second
```

Comment:
could we combine this with `defaultPublishRetryInterval`? they all set as the deadline of the timeout for internal proposal.